blockchain/transaction_confirmers.go

```diff
@@ -49,11 +50,6 @@ func NewTransactionConfirmer(client EVMClient, tx *types.Transaction, minConfirm
 // ReceiveBlock the implementation of the HeaderEventSubscription that receives each block and checks
 // tx confirmation
 func (t *TransactionConfirmer) ReceiveBlock(block NodeBlock) error {
-	if block.Block == nil {
-		// Strange case that happens in some EVM testnets
-		log.Debug().Msg("Received nil block")
-		return nil
-	}
```

Comment:
not valid anymore?

Reply:
No, we pass by values instead of pointer now, can't check for nil if we wanted it.